cmd/parse/gov/proposal.go

```diff
@@ -57,19 +80,32 @@ func proposalCmd(parseConfig *parsecmdtypes.Config) *cobra.Command {
 				return err
 			}
 
+			// Update the proposal to the latest status
+			height, err := parseCtx.Node.LatestHeight()
+			if err != nil {
+				return fmt.Errorf("error while getting chain latest block height: %s", err)
+			}
+
+			err = govModule.UpdateProposal(height, proposalID)
+			if err != nil {
+				return err
+			}
+
 			return nil
 		},
 	}
 }
 
-func refreshProposalDetails(parseCtx *parser.Context, proposalID string, govModule *gov.Module) error {
+func refreshProposalDetails(parseCtx *parser.Context, proposalID uint64, govModule *gov.Module) error {
```

Comment:
actually no, because UpdateProposal only updates the status, final tally result, content, and total deposit as a whole, for existing proposal. However, the methods below will handle the proposal submission and individual vote/individual deposit, etc.